packages/asc-ui/src/components/Checkbox/Checkbox.tsx

```diff
@@ -7,15 +7,23 @@ import CheckboxStyle, {
 } from './CheckboxStyle'
 import LabelContext from '../Label/LabelContext'
 
-const Checkbox: React.FC<Props & React.HTMLAttributes<HTMLInputElement>> = ({
-  checked: checkedProp,
-  className,
-  onChange,
-  variant,
-  disabled,
-  error,
-  ...otherProps
-}) => {
+type CheckboxProps = Props & React.HTMLAttributes<HTMLInputElement>
+
+const Checkbox: React.ForwardRefRenderFunction<
```

Comment:
We could just wrap the whole component here in a call to `forwardRef` so we can avoid having to pass the types twice.

```ts
const Checkbox = React.forwardRef<
  HTMLInputElement,
  Props & React.HTMLAttributes<HTMLInputElement>
>((
  {
    checked: checkedProp,
    className,
    onChange,
    variant,
    disabled,
    error,
    ...otherProps
  },
  ref,
) => {
  // ...
})

export default Checkbox
```

Reply:
Agreed, this is more compact 👍

Reply:
gedaan